clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -278,6 +278,11 @@
                                         Importance.MEDIUM,
                                         CommonClientConfigs.CLIENT_DNS_LOOKUP_DOC)
                                 .define(GROUP_ID_CONFIG, Type.STRING, null, Importance.HIGH, GROUP_ID_DOC)
+                                .define(GROUP_INSTANCE_ID_CONFIG,
+                                        Type.STRING,
+                                        JoinGroupRequest.EMPTY_GROUP_INSTANCE_ID,
```

Comment:
I think `null` default value should still work, e.g. the group-id used `null` as default values above.

Reply:
The tricky thing is that we couldn't set the key to `null` in `VerifiableConsumer` because it will throw exception.

Reply:
My bad, didn't see the upper comment.